service/loggingService/service.go

```diff
@@ -0,0 +1,28 @@
+package loggingservice
+
+import (
+	"fmt"
+
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/net/context"
+)
+
+type ContextKey string
+
+const ContextKeyRequestID ContextKey = "requestID"
+
+type LoggingService struct {
+}
+
+func New() LoggingService {
+	return LoggingService{}
+}
+
+func (h LoggingService) WriteLog(ctx context.Context, message string) {
```

Comment:
Function signature looks like it takes ctx not to log it but for cancelation.

Reply:
what should i do with this comment?